src/store/themeModule.js

```diff
@@ -0,0 +1,15 @@
+import * as mutationTypes from './mutationTypes';
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
There is no need to disable `import/prefer-default-export`:
```js
export default  {
  state: {}
  // ...
}
```
later in `store/index.js`
```js
import themeModule from './themeModule';
```
---
Or as an alternative you can import everything,e .g.

```js
export const state  = {
  preferDarkTheme: null,
}

export const mutations = {
  [mutationTypes.SAVE_PREFER_DARK_THEME](state, preferDarkTheme) {
      state.preferDarkTheme = preferDarkTheme;
  },
};
```
later in `store/index.js`
```js
import * as themeModule from './themeModule';
```